pkg/nsx/client.go

```diff
@@ -41,12 +64,14 @@ func GetClient(cf *config.NSXOperatorConfig) *Client {
 	groupClient := domains.NewGroupsClient(restConnector(cluster))
 	securityClient := domains.NewSecurityPoliciesClient(restConnector(cluster))
 	ruleClient := security_policies.NewRulesClient(restConnector(cluster))
+	nsxChecker := newNSXHealthChecker(cluster)
 	return &Client{
 		NsxConfig:      cf,
 		RestConnector:  restConnector(cluster),
 		QueryClient:    queryClient,
 		GroupClient:    groupClient,
 		SecurityClient: securityClient,
 		RuleClient:     ruleClient,
+		NSXChecker:     *nsxChecker,
```

Comment:
it seems no need to have a function newNSXHealthChecker(), it's only used in Line 67.
nsxChecker is not needed.
Line 75 -> NSXChecker:     NSXHealthChecker{cluster: c},